examples/form/zipcodes.csv

```diff
@@ -0,0 +1,4431 @@
+RecordNumber,Zipcode,City,State

```

Comment:
The original title of the issue was _forms: zip code input validation_, but [it was changed](https://github.com/ampproject/amphtml/issues/8369#event-1040976739). I thought reading through real California zip codes and cities would be an interesting demo. If you think it's too heavy weight we can just change it to a smaller dict.